packages/contracts-bedrock/justfile

```diff
@@ -86,6 +86,9 @@ snapshots-check:
 interfaces-check-no-build:
   ./scripts/checks/check-interfaces.sh
 
+# Checks interface correctness without cleaning before building.
+interfaces-check-no-clean: build interfaces-check-no-build
```

Comment:
This is so we can leverage the cache and only do a partial rebuild to reduce time to check interfaces?

Reply:
Yes, cleaning is useful by default but once you've cleaned you usually just need to do a rebuild